zebra-chain/src/lib.rs

```diff
@@ -27,6 +27,16 @@ pub use redjubjub;
 #[cfg(test)]
 use proptest_derive::Arbitrary;
 
+use serde_big_array::big_array;
+
+/// The size of an Equihash solution in bytes (always 1344).
+const EQUIHASH_SOLUTION_SIZE: usize = 1344;
```

Comment:
Should we use the const from equihash_solution instead?

Reply:
Oh, do you mean move it back to the `equihash_solution` module and make it pub(crate) and use it's full path in `serde_helpers` when referencing it?